js/fetch_test.ts

```diff
@@ -1,9 +1,20 @@
 // Copyright 2018 the Deno authors. All rights reserved. MIT license.
-import { testPerm, assertEqual } from "./test_util.ts";
+import { test, testPerm, assert, assertEqual } from "./test_util.ts";
 import * as deno from "deno";
 
 testPerm({ net: true }, async function fetchJsonSuccess() {
   const response = await fetch("http://localhost:4545/package.json");
   const json = await response.json();
   assertEqual(json.name, "deno");
 });
+
+test(async function fetchJsonNetPerm() {
+  let err;
+  try {
+    fetch("http://localhost:4545/package.json");
+  } catch (err_) {
```

Comment:
@ry Any idea why this catch statement does not work?

Reply:
@qti3e The `await` in front of the `fetch(...)` call is missing.

Reply:
@MarkTiedemann I tried that before...